.github/workflows/link-to-checklist.yml

```diff
@@ -0,0 +1,30 @@
+name: link-to-checklist
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  add-review-checklist:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Add review checklist
+        uses: actions/github-script@v6
```

Comment:
Maybe try updating this to v7, it looks like that’s the latest version of this action? https://github.com/actions/github-script